source/draw/gpu/opengl/GraphicBufferYuv420Semiplanar.ooc

```diff
@@ -54,7 +54,7 @@ GraphicBufferYuv420Semiplanar: class extends RasterYuv420Semiplanar {
 		result coordinateSystem = this coordinateSystem
 		result
 	}
-	kean_draw_gpu_android_graphicBufferYuv420Semiplanar_new: unmangled static func (backend: Pointer, nativeBuffer: Pointer, handle: Pointer, size: IntSize2D, stride: Int, format: GraphicBufferFormat, uvOffset: Int) -> This {
+	kean_draw_gpu_graphicBufferYuv420Semiplanar_new: unmangled static func (backend: Pointer, nativeBuffer: Pointer, handle: Pointer, size: IntSize2D, stride: Int, format: GraphicBufferFormat, uvOffset: Int) -> This {
```

Comment:
I think `backend: Pointer, nativeBuffer: Pointer, handle: Pointer` can be simplified